apps/verify/models/write-cookie.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const dataService = require('../../common/data-service');
+const {
+  makeRequest,
+  requestHeaders,
+  } = dataService;
+
+/**
+ * API post - create cookie in the database
+ *
+ * @param {object} data - {created, uuid, userEmail} sent to the db
+ *
+ * @returns {Promise} response
+ */
+const sendDataToBeStored = async(data) => {
+  // current date
+  const created = (new Date()).toISOString();
+  const uuid = data.uuid;
+  const userEmail = data['user-email'];
+
+  const body = {
+    'created': created,
+    'uuid': uuid,
+    'useremail': userEmail
+  };
+
+  const options = {
+    method: 'post',
+    body: JSON.stringify(body),
+    headers: requestHeaders,
+  };
+
+  try {
+    const response = await makeRequest('/cookies', options);
+    if (response) {
+      return uuid;
+    }
```

Comment:
if there's no `response` it'll return `undefined` -- is this ok?

Reply:
see below, does that make sense @zuzak ?